### PR TITLE
Update ht.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added a couple channel

### CODE REVIEW

File: channels/ht.m3u [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCcEY2-g-EEefxyYa1gtrk4g/live
+#EXTINF:-1 tvg-id="KajouTV.ht" tvg-country="US;HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/5txJUZx.jpg" group-title="",Kajou TV (720p)
+https://video1.getstreamhosting.com:1936/8055/8055/playlist.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/OJbRkQ6.jpg" group-title="",NETALKOLE TV (180p)
+https://watch.haitilive.net/stream/netalkole/public/tv/index.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://uni01rtmp.tulix.tv/4vehtv/4vehtv-firetv/playlist.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UClL_aynC_SESNLQEJHqO9MQ/live
```
Comment (freearhey) [inline channels/ht.m3u:11]: Offline

File: channels/ht.m3u [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="RadioTeleAmen.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/dJvJGDg.jpg" group-title="Religious",Radio Tele Amen FM (360p)
+http://184.173.179.163:1935/daniel/daniel/playlist.m3u8
```
Comment (freearhey) [inline channels/ht.m3u:13]: Offline

File: channels/ht.m3u [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="RadioTéléEclair.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/8ocCyo7.png" group-title="",Radio Télé Eclair (1080p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCOvGbDh5biuOqjx6G3CdrQg/live
```
Comment (freearhey) [inline channels/ht.m3u:15]: Offline

File: channels/ht.m3u [hunk 4]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/QaWwNvo.jpg" group-title="Religious",Radio Télé Pitit Manman Mari (720p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCLeNHM8XDkZmd2rhV3ZG7Vg/live
```
Comment (freearhey) [inline channels/ht.m3u:17]: Offline

File: channels/ht.m3u [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="RTVC.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/Jp38Rzf.png" group-title="",Radio Télévision Caraïbes (RTVC) (720p)
+https://stream-02.nyc.dailymotion.com/sec(JtO5HO8PrOFTKKYNhfGSpkCAlv5YxyPWEpr2He2oJRA)/dm/3/x81lgjh/d/live-2.m3u8
+#EXTINF:-1 tvg-id="RadioTélévisionHirondelle.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/2gVnoSX.jpg" group-title="",Radio Télévision Hirondelle (480p)
+https://watch.haitilive.net/gethb/rtvh16/index.m3u8
+#EXTINF:-1 tvg-id="RadioTeleShalom.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/MSgSKym.png" group-title="",Radio Tele Shalom (720p) [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCBJ9zxns1hxblYZw4urBd_w/live
+#EXTINF:-1 tvg-id="RadioTelePlanetCompas.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/ddb4lsa.png" group-title="",Radio Tele Planet Compas (720p)
+https://5dcab9aed5331.streamlock.net/mrcompas1/livestream/playlist.m3u8
```
Comment (freearhey) [inline channels/ht.m3u:25]: Offline

File: channels/ht.m3u [hunk 6]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="TCH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/6LZVA4x.png" group-title="",Télé Conscience Haïtienne (TCH) (480p)
+https://tvlakay.haitilive.net/website/tchhaiti/player/mono.m3u8
```
Comment (freearhey) [inline channels/ht.m3u:27]: image

File: channels/ht.m3u [hunk 7]
```diff
@@ -1,7 +1,33 @@
...
+#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://www.newsfromhaiti.com/images/TeleHaiti.jpg" group-title="General",Tele Haiti (1088p) [Offline]
 http://66.175.238.147:1935/live/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="TeleLouange.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/vZmlq9N.jpg" group-title="",Tele Louange (720p)
+https://5790d294af2dc.streamlock.net/8124/8124/chunklist.m3u8
+#EXTINF:-1 tvg-id="TVPanou.ht" tvg-country="US;HT" tvg-language="English" tvg-logo="https://i.imgur.com/p5GjWnu.jpg" group-title="",TV Panou (720p)
+http://tvpanoucom.srfms.com:1935/tvpanoucom/livestream/playlist.m3u8
```
Comment (freearhey) [inline channels/ht.m3u:33]: Offline

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Comment by Sphinxroot** [comment]
added [Not 24/7] for morst of them  

TV Panou  ,Radio Tele Planet Compas  , the are working  now cant tell if its not 24/7 , i will check them

and do we remove Télé Conscience Haïtienne (TCH)  or we let it there ?

File: channels/ht.m3u (hunk 6 above)
Comment (freearhey) [reply channels/ht.m3u:27]: The error will disappear if you change the protocol:

```suggestion
http://tvlakay.haitilive.net/website/tchhaiti/player/mono.m3u8
```

File: channels/ht.m3u [hunk 8]
```diff
@@ -1,7 +1,33 @@
...
+https://video1.getstreamhosting.com:1936/8055/8055/playlist.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/OJbRkQ6.jpg" group-title="",NETALKOLE TV (180p)
+https://watch.haitilive.net/stream/netalkole/public/tv/index.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://uni01rtmp.tulix.tv/4vehtv/4vehtv-firetv/playlist.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p) [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UClL_aynC_SESNLQEJHqO9MQ/live
+#EXTINF:-1 tvg-id="RadioTeleAmen.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/dJvJGDg.jpg" group-title="Religious",Radio Tele Amen FM (360p) [Not 24/7]
+http://184.173.179.163:1935/daniel/daniel/playlist.m3u8
```
Comment (Dum4G) [inline channels/ht.m3u:13]: image
nice

File: channels/ht.m3u [hunk 9]
```diff
@@ -1,7 +1,33 @@
...
-#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-country="HT" tvg-language="French" tvg-logo="https://www.newsfromhaiti.com/images/TeleHaiti.jpg" group-title="General",TeleHaiti (1088p) [Offline]
+#EXTINF:-1 tvg-id="HaitiViralNews.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/eaPFJJs.jpg" group-title="",Haiti Viral News (720p) [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCcEY2-g-EEefxyYa1gtrk4g/live
+#EXTINF:-1 tvg-id="KajouTV.ht" tvg-country="US;HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/5txJUZx.jpg" group-title="",Kajou TV (720p)
+https://video1.getstreamhosting.com:1936/8055/8055/playlist.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/OJbRkQ6.jpg" group-title="",NETALKOLE TV (180p)
+https://watch.haitilive.net/stream/netalkole/public/tv/index.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://uni01rtmp.tulix.tv/4vehtv/4vehtv-firetv/playlist.m3u8
```
Comment (Dum4G) [inline channels/ht.m3u:9]: I'll aprove it as is for now but tulix.tv can have it's separate playlist file @freearhey

**Review by Dum4G — APPROVED** [review]

File: channels/ht.m3u (hunk 8 above)
Comment (Sphinxroot) [reply channels/ht.m3u:13]: it come from here http://www.centraltv.fr/television-haiti/radio-tele-amen-fm

**Review by freearhey — APPROVED** [review]

**Pull request merged**